resources/views/layouts/simple.blade.php

```diff
@@ -5,7 +5,7 @@
 	<meta name="viewport"
 			content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
 
-	<title>{{ enpii_base_wp_app_web_page_title() }}</title>
+	<title><?php echo \Enpii_Base\App\Support\Enpii_Base_Helper::enpii_base_wp_app_web_page_title() ?></title>
```

Comment:
We need to keep the function `enpii_base_wp_app_web_page_title` but inside we just use `\Enpii_Base\App\Support\Enpii_Base_Helper::enpii_base_wp_app_web_page_title()` to keep the simplicity